models.py

```diff
@@ -29,71 +31,78 @@ def generate_compiled_segmentation_model(model_name, model_parameters, num_class
+    # make first metric a copy of loss, to continually verify `val_loss` is correct
     if isinstance(loss_fn, BinaryCrossentropyL):
-        all_metrics.append(BinaryCrossentropyM())
+        all_metrics.append(BinaryCrossentropyM(name='binary_ce_metric'))
     else:
-        all_metrics.append(CategoricalCrossentropyM())
-    all_metrics[0].name = str(all_metrics[0].name + '_keras_metric')
+        all_metrics.append(CategoricalCrossentropyM(name='categ_ce_metric'))
 
     for class_num in range(num_classes + 1):
         if class_num == 0:    # all class metrics
             # note, `loss_fn` for all classes placed before `all_metrics` in lineup of command window metrics and plots
             all_metrics.extend([
-                                CategoricalCELoss(),
-                                Accuracy(),
-                                OneHotAccuracy(),
-                                OneHotClassBinaryAccuracyKeras(),
-                                OneHotClassBinaryAccuracySM(),
-                                CategoricalAccuracy(),
-                                FalseNegatives(name='false_neg', thresholds=global_threshold),
-                                OneHotFalseNegatives(name='false_neg_1H'),
-                                TrueNegatives(name='true_neg', thresholds=global_threshold),
-                                OneHotTrueNegatives(name='true_neg_1H'),
-                                FalsePositives(name='false_pos', thresholds=global_threshold),
-                                OneHotFalsePositives(name='false_pos_1H'),
-                                TruePositives(name='true_pos', thresholds=global_threshold),
-                                OneHotTruePositives(name='true_pos_1H'),
-                                Recall(name='recall', thresholds=global_threshold),
-                                OneHotRecall(name='recall_1H'),
-                                Precision(name='precision', thresholds=global_threshold),
-                                OneHotPrecision(name='precision_1H'),
-                                FBetaScore(name='f1_score', beta=1, thresholds=global_threshold),
-                                OneHotFBetaScore(name='f1_score_1H', beta=1),
-                                IoUScore(name='iou_score', thresholds=global_threshold),
-                                OneHotIoUScore(name='iou_score_1H')
-                                ])
+                CategoricalCELoss(),
+                AccuracyTfKeras(),
+                OneHotAccuracyTfKeras(),
+                ClassBinaryAccuracyTfKeras(),
+                OneHotClassBinaryAccuracyTfKeras(),
+                ClassBinaryAccuracySM(),
+                OneHotClassBinaryAccuracySM(),
+                BinaryAccuracy(),
+                CategoricalAccuracy(),
+                FalseNegatives(name='false_neg', thresholds=global_threshold),
+                OneHotFalseNegatives(name='false_neg_1H'),
```

Comment:
Apologies if I'm missing it but did we also say something about either explaining why we have both the non-1h metrics and the 1h metrics or removing the 1h ones?

Reply:
@Josh-Joseph We decided to keep them to have the standard thresholded version (default threshold is 0.5), in case that would be more applicable down the road.

Reply:
okay cool -- is there text floating around in a comment somewhere explaining that?

Reply:
nope will add that

Reply:
done